packages/corvid-local-server/src/socket-api/index.js

```diff
@@ -48,9 +48,10 @@ const initServerApi = (
   const isEditorConnected = () =>
     Object.keys(editorServer.io.sockets.connected).length > 0;
 
-  const getSiteDocument = () => {
-    return localSite.getSiteDocument();
-  };
+  const getSiteDocument = () =>
+    localSite
+      .getSiteDocument()
+      .catch(e => notifyAdmin("fatal-error", e.message));
```

Comment:
this does not report the error to the editor.
if we revert this change, won't the editor catch the error and report it to the cli by itself ?